src/main/java/com/prgrms/amabnb/oauth/api/OAuthController.java

```diff
@@ -0,0 +1,32 @@
+package com.prgrms.amabnb.oauth.api;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.prgrms.amabnb.common.security.jwt.util.AuthorizationExtractor;
+import com.prgrms.amabnb.oauth.dto.AccessTokenResponse;
+import com.prgrms.amabnb.oauth.dto.RefreshTokenRequest;
+import com.prgrms.amabnb.oauth.service.TokenService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+public class OAuthController {
+
+    private final TokenService tokenService;
+    
+    @PostMapping("/token")
+    public ResponseEntity<AccessTokenResponse> refreshAccessToken(
+        HttpServletRequest httpServletRequest,
+        @RequestBody RefreshTokenRequest refreshToken
```

Comment:
DTO 내에서 validation을 해주고 있네요! 
```suggestion
        @Valid @RequestBody RefreshTokenRequest refreshToken
```

Reply:
깜빡했네요 😂